apps/egghead/src/lib/events-query.ts

```
@@ -0,0 +1,1153 @@
+
+	if (!parsedEvents.success) {
+		console.error('Error parsing events', events)
+		return []
+	}
+	if (!parsedEvents.success) {
+		console.error('Error parsing events', events)
+		return []
+	}
+
+	return parsedEvents.data
+	return parsedEvents.data
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)